### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by evelynhathaway**
I made the logo right aligned, switched to [badgen](https://github.com/amio/badgen-service) badges (as both the Travis and shields.io badges load slower), and made some minor grammar changes.

### CODE REVIEW

**Comment by evelynhathaway** [comment]
I noticed the badge on the `master` branch is a Travis badge for the `master` branch only so that may have to be changed separately.

File: README.md [hunk 1]
```diff
@@ -45,7 +46,7 @@ v.getPalette().then((palette) => console.log(palette))
 The default browser entry for `node-vibrant` has web worker support. You should make sure that `worker-loader` is installed:
 
 ```
-$ npm install --save-dev worker-loader
```
Comment (crutchcorn) [inline README.md]: Let's revert these. I like being able to easily copy+paste too but there's arguments to be made to make copy+paste commands harder and being able to easily communicate what this line is for at a glance as syntax highlighting for commands isn't easy in markdown

Comment (evelynhathaway) [reply README.md]: Gotcha. 😄

Comment (evelynhathaway) [reply README.md]: Reverted and squashed! I also added `bash` to the last shell code block.

**Comment by crutchcorn** [comment]
Thanks for the merge. We're doing some docs cleanup/etc and then `master` will be based on `develop` branch. No need to make PR for that one

**Comment by evelynhathaway** [comment]
I figured as much and didn't want to create a PR entirely to make conflicts either haha. Let me know if I can help with docs in any way. 😄

**Pull request merged**